lib/costumer.rb

```diff
@@ -0,0 +1,43 @@
+require 'csv'
```

Comment:
Spelling note: a "costumer" is someone who makes costumes, like for a movie studio. You're probably thinking of a "customer" here.